index.html

```diff
@@ -0,0 +1,96 @@
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-navbar">
+            <span class="menu" id="menu"><i class="fas fa-bars"></i></span>
+            <a href="index.html"><img class="logo" src="./images/logo.png" alt="logo"></a>
+            <ul class="second-list" id="mobile-menu">
+                <li><i class="fas fa-times close-btn" id="close-btn"></i></li>
+                <li><a href="about.html" class="nav-list" id="about-link">About</a></li>
+                <li><a href="#" class="nav-list">Program</a></li>
+                <li><a href="#" class="nav-list">Join</a></li>
+                <li><a href="#" class="nav-list">Sponsor</a></li>
+                <li><a href="#" class="nav-list">News</a></li>
+                <li><a href="#" class="button nav-list">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="main-container">
+        <h1 class="main-heading">Hello! Creative world</h1>
+        <h2 class="main-sec-heading">CREATIVE CULTURE GLOBAL SUMMIT 2015</h2>
+        <p class="main-par">Join the world's brightest minds, innovators, and visionaries for an immersive exploration of the limitless 
+            possibilities at the Creative Culture Global Summit. Ignite your creativity, nurture cross-cultural connections,
+             and shape the future of artistic expression. Together, let's redefine boundaries and unlock the true potential 
+             of the global creative community. Get ready to embark on a transformative journey, where ideas converge and a 
+             new era of cultural innovation begins in October, in Ghana. </p>
+        <p class="bold">
+            2015.10.15(THU)~16(FRI)
+        </p>
+        <p class="bottom-text">@ The Accra International Conference Centre, Accra, Ghana</p>
+    </section>
+
+    <section class="main-program" id="program">
+        <div class="wrapper">
+            <div class="program-container">
+                <div class="program-title flex-container">
+                    <h3 class="main-section-h1">Main Program</h3>
+                    <div class="orange-underline"></div>
+                </div>
+                <div class="program-categories">
+                </div>
+                <a class="btn-mobile-program">SEE THE WHOLE PROGRAM</a>
+                <button type="button" class="btn-program">Join CC Summit 2015</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="speaker">
+        <h2 class="main-section-h1 ">Featured Speakers</h2>
+        <div class="orange-underline"></div>
+        <div class="speaker-container" id="container">
+        </div>
+        <button type="button" class="more-btn">More <img src="./images/arrow_down.png" alt="arrow_down"></button>
+    </section>
```

Comment:
- [ ] You did a great job following the previous reviewer's advice on modifying the content and styles of your speaker section! 💯 However, a new bug was introduced:

![image](https://github.com/erickma1/CapstoneProject/assets/77069011/ddccd0db-230b-4aa4-be09-740ecb35780a)

There's horizontal scrolling which means that some content is overflowing the width of the device and there isn't spacing to the edges as suggested by the design:

![image](https://github.com/erickma1/CapstoneProject/assets/77069011/c32900fe-e2fb-4e9a-9fca-0a71c907d916)

The horizontal scrolling is happening because you set a fixed width of `800px` in your speaker card. Please consider fixing both of these requirements! 🤓 🔧 


- [ ] Something similar is happening in the mobile version:

|Template|Your design|
|---|---|
| ![image](https://github.com/erickma1/CapstoneProject/assets/77069011/f0cd24bc-6080-4041-9ea0-61a35cb93440) | ![image](https://github.com/erickma1/CapstoneProject/assets/77069011/52a6c4ac-d0bb-4920-9a9c-65aa244c170c) |

If you see, the upper sections have empty blank space to the right and there's horizontal scrolling.


Please consider fixing this! 🤓 🔧